HW_7/task_34.py

```diff
@@ -0,0 +1,17 @@
+input_text = input("Write your song sounds: ")
+check_letters = list('аеёиоуыэюя')
+lett_list = []
+# input_text.split(" ")
+
+
+for each in input_text.split(" "):
+    filtered = filter(lambda x: x in check_letters, each)
+    let_amount = len(list(map(lambda x: x, filtered)))
+    lett_list.append(let_amount)
```

Comment:
А зачем 9 строка?
```suggestion
    filtered = len(list(filter(lambda x: x in check_letters, each)))
    lett_list.append(filtered)
```
или
```suggestion
    filtered = sum(map(lambda x: x in check_letters, each))
    lett_list.append(filtered)
```